### ISSUE TITLE
Holding forwards/backwards buttons do not work using car steering wheel controls

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#538]
> I have a Chrysler Pacifica van 2017. With other apps, I can use my steering wheel controls to skip forward and back. This doesn’t work with Podverse. Holding the steering wheel buttons for forward and reverse only seems to mute the playback. Is there something I need to do to get those working?

We should probably start by investigating react-native-track-player to see how/if other apps have handled it.

**Comment by mitchdowney** [comment]
I think this is blocked by https://github.com/DoubleSymmetry/react-native-track-player/issues/1291

**Comment by mitchdowney** [comment]
duplicate of https://github.com/podverse/podverse-rn/issues/805
